pippo-server-parent/pippo-jetty/src/main/java/ro/pippo/jetty/JettyServer.java

```diff
@@ -58,46 +55,8 @@
 
     private Server server;
 
-    private final ExecutorService executor;
-    private final CountDownLatch  startLatch;
-
-    public JettyServer() {
```

Comment:
The noarg superconstructor should be called by default: https://stackoverflow.com/a/2054040